src/bacnet/basic/npdu/h_npdu.c

```diff
@@ -194,6 +195,11 @@ static void network_control_handler(BACNET_ADDRESS *src,
         default:
             break;
     }
+    if (npdu_data->network_message_type >= NETWORK_MESSAGE_ASHRAE_RESERVED_MIN && npdu_data->network_message_type <= NETWORK_MESSAGE_ASHRAE_RESERVED_MAX)
```

Comment:
please run clang-format on this file for the changes (80 column limit will be used)
please add extra parenthesis so developer don't have to think about precedence.